gabrielaEnache/src/main/java/collections/store/User.java

```diff
@@ -0,0 +1,70 @@
+package collections.store;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class User {
+    private static long id = 0;
```

Comment:
The same as for Item class, please take a look above.

Reply:
Fixed